runtime/Cargo.toml

```diff
@@ -11,6 +11,8 @@ targets = ["x86_64-unknown-linux-gnu"]
 
 [dev-dependencies]
 cennznet-cli = { path = "../cli", default-features = false }
+sp-keyring = { git = "https://github.com/plugblockchain/plug-blockchain", default-features = false, branch = "spike/2.0" }
```

Comment:
do we need keyring here? since we use it in runtime/tests/common/keyring

Reply:
Yes correct.